packages/components/Atoms/Icon/stories.js

```diff
@@ -63,12 +70,14 @@ storiesOf('Atoms/Icon', module)
           'help',
           'home',
           'ics',
+          'image_gallery_arrow_left',
+          'image_gallery_arrow_right',
```

Comment:
Hey @alan-cole , I feel it can be fixed by css stoke, how do you think? Instead of add another svg icon just for adding border. See https://css-tricks.com/almanac/properties/s/stroke/
It may can be a scss mixin, so we can reuse it for any svg icons.

Reply:
I've implemented these changes, if you could re-check?